config/rsbuild-config-cozy-app/getRsbuildConfig.js

```diff
@@ -85,6 +85,10 @@ function getRsbuildConfig({
     },
     tools: {
       rspack: {
+        // We want to watch cozy-* packages when using rlink
+        watchOptions: {
+          ignored: /\.git|node_modules\/(?!cozy-)/
```

Comment:
so we need to name our lib with `cozy-*` 🤔 don't have any better idea right now but it's a limitation for sure... 🤔  🤔 maybe we can at least document it somewhere?

Reply:
README updated